librz/core/cmd_descs/cmd_debug.yaml

```diff
@@ -281,6 +281,10 @@ commands:
         summary: Continue until call (use step into)
         cname: cmd_debug_continue_call
         args: []
+      - name: dcco
+        summary: Continue until call (use step out)
+        cname: cmd_debug_continue_call_over_handler
```

Comment:
@gogo2464 you did it wrong. just remove `_handler` from this, rebuild and commit.